app/src/main/java/at/roteskreuz/stopcorona/model/managers/DatabaseCleanupManager.kt

```diff
@@ -65,6 +53,8 @@ class DatabaseCleanupManagerImpl(
             }
 
             val nowAsRollingStartIntervalNumber = ZonedDateTime.now()
```

Comment:
name does not match content

Reply:
Suggestion: `thresholdRevokedMessagesAsRollingStart`

Reply:
updated